drakrun/drakrun/vm.py

```diff
@@ -86,12 +138,18 @@ def restore(self) -> None:
         # is correct by definition.
         if self.vm_id != 0:
             self.backend.rollback_vm_storage(self.vm_id)
+        else:
+            if not os.path.exists(os.path.join(VOLUME_DIR, "vm-0.img")):
+                self.backend.initialize_vm0_volume(InstallInfo.try_load().disk_size)
```

Comment:
It had happened with me that there was no `vm-0.img` file and my `python3 drakplayground 0` was crashing that time, when I did this, the restore was successful from the `snapshot.sav` file. Without this, it was crashing with some errors hinting that `vm-0.img` did not exist

Reply:
If there's no `vm-0.img`, restoring it will result in imminent OS crash when it tries to read or write to disk.
Playground is more of a debugging / testing tool, and it requires installing the snapshot correctly before using it.

Reply:
> If there's no `vm-0.img`, restoring it will result in imminent OS crash when it tries to read or write to disk.

Oh okay, I will remove this